### DISCUSSION TITLE
关于fragment的几个疑问

### DISCUSSION THREAD

**Discussion by GreatMichaelLee** [XTLS/Xray-core#2430] [Q&A]
1. 我的outbound里freedom协议配了两个，一个UseIPv4, 一个UseIPv6, 用来分流普通fq和流媒体，请问fragement要在这两个freedom里分别都配吗，还是只支持ipv4?
2. 我看某些面板里在vnext节点里用的sockopt+dialproxy的方式去访问，是否必须？还是fragment用普通出站tag就OK了？
3. 配fragment是在server端配置还是client端？还是两端都要？

**Comment by chika0801** [comment]
https://github.com/XTLS/Xray-core/issues/2392

这功能你在中国大陆暂时用不到吧


> sockopt+dialproxy的方式

看你这句描述好像是俗称的链式代理形式，具体你要发完整配置才能分析。

**Comment by GreatMichaelLee** [comment]
用不到？那主要是针对什么场景开发这个？？



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年8月11日(周五) 下午5:31
收件人: ***@***.***&gt;;
抄送: "michael ***@***.******@***.***&gt;;
主题: Re: [XTLS/Xray-core] 关于fragment的几个疑问 (Discussion #2430)




 
#2392
 
这功能你在中国大陆暂时用不到吧
  
sockopt+dialproxy的方式
  
看你这句描述好像是俗称的链式代理形式，具体你要发完整配置才能分析。

**Reply by chika0801** [reply]
你去 pr 区里找找这个PR，我印象中是伊朗朋友提的，他们那边会用到。
